client/disputes/utils.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { CachedDispute } from 'wcpay/types/disputes';
+import { formatCurrency } from 'wcpay/utils/currency';
+
+interface GetDisputesNoticeStringArgs {
+	/**
+	 * Active disputes (awaiting a response) to generate the notice string for.
+	 */
+	activeDisputes: CachedDispute[];
+}
+
+export const getDisputesNoticeString = ( {
```

Comment:
Creating this function in `disputes/utils`, allowing re-use in `active-disputes-notice` and "Things to do" task item (future PR).

Reply:
Thanks for clarifying; my first question was going to be "is this going to be reused"!

If it was me I'd implement it inline and refactor for the second client (or later).

Reply:
(aka premature generalisation)

Reply:
Yeh I was borderline about this (PS I wrote this function before our discussion in the other PR #6361 so I unapologetically wrote the `utils` file before the component in a TDD fashion 😝).

I know we will have at least _some_ shared dispute logic, I leaned towards creating the `disputes/utils` file as a framework for this project, in a similar approach to the simplify deposits project.

Reply:
Although still a WIP draft, I'm working on the "Things to do" wording PR #6383 in parallel and it looks like sharing this `getDisputesNoticeString` util function will be the simplest way to implement:

https://github.com/Automattic/woocommerce-payments/pull/6383/files#diff-062eb1e5dc64b4146f2536ba4987039bedb128b2d938de7217119a9beaa2fd87R185

Reply:
I've now moved this logic within the `client/overview/task-list/dispute-tasks.ts` file, consistent with logic for other task list items.